### PR TITLE
Feat/921 change icons doc

### PR DESCRIPTION
**Pull request by veilvokay**

### CODE REVIEW

**Review by thrbnhrtmnn — COMMENTED** [review]
We should elaborate a bit more how changing default icons actually can be done.

As I understand it, there are multiple ways like for example replacing a default icon but keeping the name or by defining a new default icon within the component.

Please also get 2 other approvals, as I can not verify if the process you described to update the icon is sufficient or not.

File: README.md
Comment (angsherpa456) [inline README.md:129]: Please maintain consistency without the new lines after the headings

File: README.md
Comment (angsherpa456) [inline README.md:28]: "\" is not required, can be just :white_check_mark:

File: README.md
Comment (veilvokay) [reply README.md:129]: @angsherpa456 there is always a new line after a '##' heading

**Review by ChristianHoffmannS2 — APPROVED** [review]

**Comment by thrbnhrtmnn** [comment]
Hey @veilvokay , I am ready to approve the task, but we should merge the PR #952  from @davidken91 first. This may lead to some conflicts in your branch, as David also worked on the readme. This is why I will wait with the approval.

**Review by davidken91 — APPROVED** [review]

File: README.md
Comment (davidken91) [inline README.md:154]: Small typo here

File: README.md
Comment (davidken91) [inline README.md:214]: Small typo here too

File: README.md
Comment (davidken91) [inline README.md:235]: Small typo. Looks good to me other than the small spelling mistakes

**Comment by thrbnhrtmnn** [comment]
Hey @veilvokay , #952 has just been merged and leads to some conflicts. Once these are resolved  and we have 2 approvals again, we can also merge your branch :-)

**Review by JpunktWpunkt — APPROVED** [review]

**Review by davidken91 — APPROVED** [review]

**Pull request merged**
